.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "githubPullRequests.ignoredPullRequestBranches": [
+        "master"
+    ]
+}
```

Comment:
> I did not mean to modify/create this file `.vscode/settings.json` :frowning_face: I did this via VSCode built into Gthub and I guess that's where it came from. I will try to remove it. Or maybe someone else can!

I can remove this for you, or you can give it a shot. From the command line, you'd do something like:

```
git switch open-sourcing
git reset HEAD~ .vscode/settings.json
git commit --amend
```

Reply:
Hm, I tried this, but it doesn't seem to have done anything. After running the second line, should I see changes in `git status`? (I do not)

Reply:
Hmm. I guess I don't have a correct mental model of your git working dir then. :upside_down_face: It worked for me when I tried it before recommending it. You should indeed have seen changes in `git status` after the `git reset`.

Should I just do it?

Reply:
Ah sorry I just saw this comment. I deleted the file in another commit, but I realise that may be messy. Sorry 😬